browser/src/control/Control.NotebookbarCalc.js

```diff
@@ -1735,11 +1735,35 @@ L.Control.NotebookbarCalc = L.Control.NotebookbarWriter.extend({
 				'vertical': 'true'
 			},
 			{
-				'id': 'data-calculate',
```

Comment:
@pedropintosilva Are we really using `Recalculate` in Online? I think it is useful once we have a check menu of `AutoCalculate`.